codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Error.java

```diff
@@ -35,6 +38,18 @@
     HTTP_1_1_REQUIRED(0xD);
 
     private final long code;
+    private static final IntObjectMap<Http2Error> INT_TO_ENUM_MAP;
+    static {
+        IntObjectMap<Http2Error> map = new IntObjectHashMap<Http2Error>();
```

Comment:
@Scottmitch I think we could just use an array instead of this map since all of the IDs are sequential and only go up to 13. WDYT?